SEED-platform/seed
# Removed vulnerable, outdated dependencies

#### Any background context you want to provide?
We were seeing security alerts because nested dependencies (in some cases 6-layers deep) had known vulnerabilities, and there was no way to override them.  In addition, the main dependencies hadn't seen updates in 2-3 years and in some cases deprecated.
#### What's this PR do?
Removes grunt entirely from the dependency stack, calls protractor directly for e2e functional tests.  Once we're ready to re-incorporate code-coverage testing we should be using something like [nyc](https://www.npmjs.com/package/nyc) instead.
#### How should this be manually tested?
Run `npm install`

Travis functional tests should pass, or you can call `./bin/protractor_start_server.sh`
